src/particles/Push.cpp

```diff
@@ -71,7 +74,11 @@ namespace detail
             amrex::ParticleReal & py = m_part_py[i];
             amrex::ParticleReal & pt = m_part_pt[i];
 
-            m_element(p, px, py, pt);
+            //amrex::Print() << "Reference particle pt: " << m_ref_part.pt << std::endl;
+
+            // push through element;
```

Comment:
```suggestion
            // push through element
```

Reply:
Ok.  This line was just used for benchmarking.  However, the updated "push through element" line appears to be missing ;

Reply:
That line should still be there: red is removed and green added.

In that line, I just removed the trailing `;`